src/router.ts

```diff
@@ -238,7 +241,8 @@ v2Router.put("/:name+/manifests/:reference", async (req, env: Env) => {
     throw new ServerError("Content type not defined", 400);
   }
 
-  const { name, reference } = req.params;
+  const { reference } = req.params;
```

Comment:
can these `library` prefix logic be in the fallback client when a docker.io registry is being in use?

Reply:
niot sure where you mean the logic should be put..

Reply:
``` 
 const name = req.params.name.includes("/") ? req.params.name : `library/${req.params.name}`;
```

This could probably get moved to http.ts? Seems very specific to how docker.io does things.

Reply:
done